ramls/loan.json

```diff
@@ -0,0 +1,143 @@
+      "type": "string"
+    },
+    "proxyUserId": {
+      "description": "ID of the user representing a proxy for the patron",
+      "type": "string",
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
+    },
+    "itemId": {
+      "description": "ID of the item lent to the patron",
+      "type": "string"
+    },
+    "itemEffectiveLocationIdAtCheckOut": {
+      "description": "The effective location, at the time of checkout, of the item loaned to the patron.",
+      "type": "string",
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
+    },
+    "status": {
+      "description": "Overall status of the loan",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the status (currently can be any value, values commonly used are Open and Closed)",
+          "type": "string"
+        }
+      },
+      "additionalProperties": true
+    },
+    "loanDate": {
+      "description": "Date time when the loan began (typically represented according to rfc3339 section-5.6. Has not had the date-time format validation applied as was not supported at point of introduction and would now be a breaking change)",
+      "type": "string"
+    },
+    "dueDate": {
+      "description": "Date time when the item is due to be returned",
+      "type": "string",
+      "format": "date-time"
+    },
+    "returnDate": {
+      "description": "Date time when the item is returned and the loan ends (typically represented according to rfc3339 section-5.6. Has not had the date-time format validation applied as was not supported at point of introduction and would now be a breaking change)",
+      "type": "string"
+    },
+    "systemReturnDate" : {
+      "description": "Date time when the returned item is actually processed",
+      "type": "string",
+      "format": "date-time"
+    },
+    "action": {
+      "description": "Last action performed on a loan (currently can be any value, values commonly used are checkedout and checkedin)",
+      "type": "string"
+    },
+    "actionComment": {
+      "description": "Comment to last action performed on a loan",
+      "type": "string"
+    },
+    "itemStatus": {
+      "description": "Last item status used in relation to this loan (currently can be any value, values commonly used are Checked out and Available)",
+      "type": "string"
+    },
+    "renewalCount": {
+      "description": "Count of how many times a loan has been renewed (incremented by the client)",
+      "type": "integer"
+    },
+    "loanPolicyId": {
+      "description": "ID of last policy used in relation to this loan",
+      "type": "string"
+    },
+    "checkoutServicePointId": {
+      "description": "ID of the Service Point where the last checkout occured",
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
```

Comment:
there is a UUID schema for this checking, I think we don't have to use pattern here